yjit/src/options.rs

```diff
@@ -87,13 +99,18 @@ pub static mut OPTIONS: Options = Options {
 };
 
 /// YJIT option descriptions for `ruby --help`.
-static YJIT_OPTIONS: [(&str, &str); 8] = [
+static YJIT_OPTIONS: [(&str, &str); 11] = [
     ("--yjit-stats",                    "Enable collecting YJIT statistics"),
     ("--yjit-trace-exits",              "Record Ruby source location when exiting from generated code"),
     ("--yjit-trace-exits-sample-rate",  "Trace exit locations only every Nth occurrence"),
     ("--yjit-exec-mem-size=num",        "Size of executable memory block in MiB (default: 128)"),
-    ("--yjit-call-threshold=num",       "Number of calls to trigger JIT (default: 30)"),
+
+    ("--yjit-call-threshold=num",       "Number of calls to trigger JIT"),
+    ("--yjit-small-threshold=num",      "Call threshold for small apps (default: 30)"),
+    ("--yjit-large-threshold=num",      "Call threshold for large deployments (default: 120)"),
+    ("--yjit-large-iseq-count=num",     "Live ISEQs to consider an app large (default: 40000)"),
     ("--yjit-cold-threshold=num",       "Global call after which ISEQs not compiled (default: 200K)"),
```

Comment:
```suggestion
    ("--yjit-large-iseq-count=num",     "Live ISEQs to consider an app large (default: 40K)"),
    ("--yjit-cold-threshold=num",       "Global call after which ISEQs not compiled (default: 200K)"),
```

Reply:
I'm mildly leaning towards undoing this format so that we could refer to a constant (which we need to define and use for setting the actual default) without formatting it if we refactor it and allow these strings to be dynamic.